auth/src/main/kotlin/com/tidal/sdk/auth/storage/legacycredentials/LegacyTokens.kt

```diff
@@ -0,0 +1,33 @@
+package com.tidal.sdk.auth.storage.legacycredentials
+
+import com.tidal.sdk.auth.model.Tokens
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+internal sealed class LegacyTokens {
+    abstract fun toTokens(): Tokens
+}
+
+@Serializable
+internal data class TokensV1(
+    @SerialName("credentials")
```

Comment:
You don't need this annotation since the name of the argument is the same

Reply:
ah yes you're right. This was a leftover I forgot to remove